tests/unit/parser/_pydantic/test_appsync.py

```diff
@@ -0,0 +1,43 @@
+import json
+import pathlib
+import pytest
+
+from aws_lambda_powertools.utilities.parser import parse, ValidationError
+from aws_lambda_powertools.utilities.parser.models.appsync import AppSyncResolverEventModel
+
+
+def load_event(filename: str) -> dict:
+    """
+    Load a JSON event from the events directory.
+
+    The function navigates four levels up from the current file to locate the
+    `tests/events` folder.
+    """
+    event_path = pathlib.Path(__file__).parent.parent.parent.parent / "events" / filename
+    with event_path.open() as f:
+        return json.load(f)
```

Comment:
You can import it from `from tests.functional.utils import load_event` and remove this function.

Check: https://github.com/aws-powertools/powertools-lambda-python/blob/develop/tests/unit/parser/_pydantic/test_alb.py#L5